app/multitenant/dynamo_collector.go

```diff
@@ -451,6 +573,22 @@ func (c *dynamoDBCollector) Add(ctx context.Context, rep report.Report) error {
 		return err
 	}
 
+	// fourth, put it in memcache
+	if c.memcache != nil {
+		// REVIEWER: Should we call this "Put" to be consistent with S3, or
+		// "Set" to be consistent with memcache?
```

Comment:
ISTR you expressing an opinion on the subject. Leaving as "Put" for now & deleting the comment.